src/bitcoin-cli.cpp

```diff
@@ -58,6 +59,7 @@ static void SetupCliArgs(ArgsManager& argsman)
     argsman.AddArg("-getinfo", "Get general information from the remote server. Note that unlike server-side RPC calls, the results of -getinfo is the result of multiple non-atomic requests. Some entries in the result may represent results from different states (e.g. wallet balance may be as of a different block from the chain state reported)", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     SetupChainParamsBaseOptions(argsman);
     argsman.AddArg("-named", strprintf("Pass named instead of positional arguments (default: %s)", DEFAULT_NAMED), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-netinfo", "Get network peer connection information from the remote server. An optional boolean argument can be passed for a detailed peers listing (default: false).", ArgsManager::ALLOW_BOOL, OptionsCategory::OPTIONS);
```

Comment:
7e0727b73841d5f2abc96cd2fac50de74546a281

For Bitcoin Core it seems conventional to use `-netinfo=<...>`, no?

Reply:
Unless I'm confused, I think `-option=<...>` would be for config args, not CLI ones. See `-generate`.